README.md

```diff
@@ -28,19 +28,30 @@ Contributions are very welcome.
 
 Suggested flow:
 
- - Open an issue to discuss the fix / improvement. Might not be needed for trivial things.
- - Fork the repo and create a feature/bug-fix branch from `dev` branch. For trivial changes you might want to work directly on forked `dev` branch.
+ - Open an issue to discuss the fix / improvement. Might not be needed for 
+ trivial things.
+ - Fork the repo and create a feature/bug-fix branch from `dev` branch. 
+ For trivial changes you might want to work directly on forked `dev` branch.
  - Send a PR to `dev` branch
 
+You will find the full description on top of the [issues](issues). There are 
+also existing templates that will be used automatically when opening a new 
+issue or pull request.
+
 # Security Issues
 
-If you have identified a security issue, please email **giuseppe.mazzapica [at] gmail.com** and do not
+If you have identified a security issue, please email 
```

Comment:
About the line breaks: Now I got what this is about. To explain the _why_ with them: It's about the 80 character line length limit to avoid horizontal scrolling.